bookkeeper-benchmark/build.gradle

```diff
@@ -42,6 +42,8 @@ dependencies {
     implementation depLibs.nettyTransportNativeEpoll
     implementation depLibs.zookeeper
     implementation depLibs.log4j12api
+    implementation depLibs.log4jCore
```

Comment:
@pradeepbn Kindly consider changing it to testImplementation

Reply:
This is moved to separate PR #2877